pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -453,6 +453,17 @@ def get_bestnrs(trigs, q=4.0, n=3.0, null_thresh=(4.25, 6), snr_threshold=6.,
     return bestnr
 
 
+# =============================================================================
+# Veto trigger
+# =============================================================================
+def veto_trig(trigs, trial_dict):
+    """Remove trigs that are not in trial dict"""
+    mask = []
+    for time in trigs['end_time']:
+        mask.append(any([time in trial for trial in trial_dict]))
+    return {key: trigs[key][mask] for key in trigs.keys()}
```

Comment:
This will not work because we `trigs.keys()` is `<KeysViewHDF5 ['H1', 'L1', 'V1', 'network']>`, so we want to apply `[mask]` a level deeper in the hierarchy of keys.